components/brave_vpn/browser/connection/win/utils.cc

```diff
@@ -344,13 +383,17 @@ bool CreateEntry(const std::wstring& entry_name,
   DWORD dw_ret = RasSetEntryProperties(DEFAULT_PHONE_BOOK, entry_name.c_str(),
                                        &entry, entry.dwSize, NULL, NULL);
   if (dw_ret != ERROR_SUCCESS) {
-    PrintRasError(dw_ret);
-    return false;
+    result.error_description = PrintRasError(dw_ret);
+    result.success = false;
```

Comment:
maybe we can create a wrapper to be like: `return RasSuccessResult()` or `return RasErrorResult(code)` instead of multiple calls in this file lilke:
```
    Result result;
    result.error_description = PrintRasError(dw_ret);
    result.success = false;
```